src/Integration.UnitTests/Connection/ConnectionWorkflowTests.cs

```diff
@@ -66,7 +66,7 @@ public void ConnectionWorkflow_ConnectionStep_SuccessfulConnection()
             var projects = new ProjectInformation[] { new ProjectInformation { Key = "project1" } };
             this.sonarQubeService.ReturnProjectInformation = projects;
             bool projectChangedCallbackCalled = false;
-            ConnectedProjectsCallback projectsChanged = (c, p) =>
```

Comment:
Removed this delegate sonce SetProject could be called directly (and tested directly)